File: Data Types and Variables/Exercises/Vowel or Digit.cs
using System;
using System.Globalization;
using System.Text;
using static System.Console;

class Program
{
    static void Main()
    {
        char input = char.Parse(ReadLine());

        if(input>='0'&&input<='9')
            WriteLine("digit");
        else if (input == 'a' || input == 'e' || input == 'i' || input == 'o' || input == 'u'||
                 input == 'A' || input == 'E' || input == 'I' || input == 'O' || input == 'U')
                WriteLine("vowel");
        else
            WriteLine("other");
    }
}

File: Software Technologies/Java/Exercises/Vowel or Digit.java
package com.company;
import java.util.*;

public class Main {
    static Scanner Input = new Scanner(System.in);

    public static void main(String[] args){
        char letter = Input.next().charAt(0);
        String output;
        if(isVowel(letter))
            output = "vowel";
        else if(Character.isDigit(letter))
            output = "digit";
        else
            output = "other";

        System.out.println(output);
    }

    private static boolean isVowel(char letter) {
        ArrayList<Character> vowels = new ArrayList<>();
        vowels.add('a');
        vowels.add('e');
        vowels.add('i');
        vowels.add('o');
        vowels.add('u');
        return vowels.contains(letter);
    }
}